src/mame/drivers/itt3030.cpp

```diff
@@ -278,165 +279,11 @@ class itt3030_state : public driver_device
-	for(int y = 0; y < 24; y++ )
-	{
-		int vramy = (start + y) % 24;
-		for(int x = 0; x < 80; x++ )
-		{
-			uint8_t code = m_vram[x + vramy*128];
-			int invert = code & 0x80 ? 1 : 0;
-			code &= 0x7f;
-			m_gfxdecode->gfx(invert)->opaque(bitmap,cliprect,  code , 0, 0,0, x*8,y*12);
-		}
-	}
-
-	return 0;
-}
-
-WRITE_LINE_MEMBER(itt3030_state::fdcirq_w)
-{
-	m_fdc_irq = state;
-}
-
-#include "debugger.h"
-
-WRITE_LINE_MEMBER(itt3030_state::fdcdrq_w)
-{
-	m_fdc_drq = state;
-}
-
-WRITE_LINE_MEMBER(itt3030_state::fdchld_w)
-{
-	m_fdc_hld = state;
-}
-
-/*
-    7 Data Request (DRQ - inverted 1791-Signal)
-    6 Interrupt Request (INTRQ - 1791-Signal)
-    5 Head Load (HLD - inverted 1791-Signal)
-    4 Ready 3 (Drive 3 ready)
-    3 Ready 2 (Drive 2 ready)
-    2 Ready l (Drive 1 ready)
-    1 Write protect (the disk in the selected drive is write protected)
-    0 HLT (Halt signal during head load and track change)
-*/
-READ8_MEMBER(itt3030_state::fdc_stat_r)
-{
-	uint8_t res = 0;
-	floppy_image_device *floppy1 = m_con1 ? m_con1->get_device() : nullptr;
-	floppy_image_device *floppy2 = m_con2 ? m_con2->get_device() : nullptr;
-	floppy_image_device *floppy3 = m_con3 ? m_con3->get_device() : nullptr;
-
-	res = m_fdc_drq ? 0x80 : 0x00;
-	res |= m_fdc_irq ? 0x40 : 0x00;
-	res |= m_fdc_hld ? 0x00 : 0x20;
-	if (floppy3) res |= !floppy3->ready_r() ? 0x10 : 0;
-	if (floppy2) res |= !floppy2->ready_r() ? 0x08 : 0;
-	if (floppy1) res |= !floppy1->ready_r() ? 0x04 : 0;
-	if (m_curfloppy) res |= m_curfloppy->wpt_r() ? 0x02 : 0;
-
-	return res;
-}
-
-/* As far as we can tell, the mess of ttl de-inverts the bus */
-READ8_MEMBER(itt3030_state::fdc_r)
-{
-	return m_fdc->gen_r(offset) ^ 0xff;
-}
-
-WRITE8_MEMBER(itt3030_state::fdc_w)
-{
-	m_fdc->gen_w(offset, data ^ 0xff);
-}
-
-/*
-    7 SEL1 - Select drive 1
-    6 SEL2 - Select drive 2
-    5 SEL3 - Select drive 3
-    4 MOTOR - Motor on
-    3 DOOR - Drive door lock drives 1 + 2 (not possible with all drives)
-    2 SIDESEL - Select disk side
-    1 KOMP - write comp on/off
-    0 RG J - Change separator stage to read
-*/
-WRITE8_MEMBER(itt3030_state::fdc_cmd_w)
-{
-	floppy_image_device *floppy = nullptr;
-
-	logerror("%02x to fdc_cmd_w: motor %d side %d\n", data, (data & 0x10)>>4, (data & 4)>>2);
-
-	// select drive
-	if (data & 0x80)
-	{
-		floppy = m_con1 ? m_con1->get_device() : nullptr;
-	}
-	else if (data & 0x40)
-	{
-		floppy = m_con2 ? m_con2->get_device() : nullptr;
-	}
-	else if (data & 0x20)
-	{
-		floppy = m_con3 ? m_con3->get_device() : nullptr;
-	}
-
-	// selecting a new drive?
-	if (floppy != m_curfloppy)
-	{
-		m_fdc->set_floppy(floppy);
-		m_curfloppy = floppy;
-	}
-
-	if (floppy != nullptr)
-	{
-		// side select
-		floppy->ss_w((data & 4) ? 1 : 0);
-
-		// motor control (active low)
-		floppy->mon_w((data & 0x10) ? 0 : 1);
-	}
-}
+//**************************************************************************
+//  ADDRESS MAPS - Alphatronic P1, P2, P2S, P2U and Hell 2069
```

Comment:
It looks like there are a bunch of comments referring to the Alphatronic machines in here when this is the itt3030 source file - bad copy/pasta?

Reply:
Thanks for catching this ... was carrying over the labels and forgot to remove the  comments.